client/state/reducer.js

```diff
@@ -186,8 +184,4 @@ const reducers = {
 	wordads,
 };
 
-if ( config.isEnabled( 'mailchimp' ) ) {
```

Comment:
This was disabling `mailchimp` reducer when running in Desktop or Horizon, and this check is lost now.
Probably not important though.